src/Microsoft.Framework.Caching.SqlServer/Program.cs

```diff
@@ -0,0 +1,55 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+using System.Data;
+using System.Data.SqlClient;
+
+namespace Microsoft.Framework.Caching.SqlServer
+{
+    public class Program
+    {
+        private string _connectionString;
+        private string _tableName;
+        private SqlQueries _sqlQueries;
+
+        public void Main(string[] args)
+        {
+            //TODO1: use CommandLine and error checks
+            _connectionString = args[0];
```

Comment:
Is it ok to require customers to provide the whole connection string here or could this be just a configuration key? If we decide to do the latter we should probably find a common way to do it across frameworks (or a convention that we all agree is ok, e.g. looking only at secrets and config.json).